src/lib/room.js

```diff
@@ -14,6 +14,10 @@ const commands = new Commands();
 
 export const closeChat = async () => {
 	await handleTranscript();
+	route('/survey-feedback');
```

Comment:
I don't like to code that. I prefer to implement a new configuration on the backend side that defines whether the `Survey Feedback` feature is enabled or not.

Let's think about it.

Reply:
Hi @renatobecker,
I like this idea. This would be like the `Transcript` feature which Livechat already has. I think the admin should have a setting at the main server, to On/Off the Feedback section. This would be more powerful.

Reply:
> Hi @renatobecker,
> I like this idea. This would be like the `Transcript Chats` feature which Livechat already has. I think the admin should have a setting at the main server, to On/Off the Feedback section. This would be more powerful.

Yeah, that's the idea, we gonna provide a setting to enable/disable the survey feedback.

Reply:
Hello, 
Added a PR in the main server which will add a setting to show/hide a Feedback form. Also updated the code here to connect with the backend setting.
Link to PR - https://github.com/RocketChat/Rocket.Chat/pull/17838